### PR TITLE
Update handlebars to v4.5.3

### PR DESCRIPTION
**Pull request by jaredclarke**
deps: handlebars@4.5.3
    - Fix security issues 
    - https://www.npmjs.com/advisories/1300
    - https://www.npmjs.com/advisories/1316
    - https://www.npmjs.com/advisories/1324
    - https://www.npmjs.com/advisories/1325

### CODE REVIEW

**Comment by andreeatirgovatu** [comment]
Hi. This upgrade is very important. It appears as a high vulnerability. When it will be merged? Thank you!

**Comment by DanielRuf** [comment]
Would be good to have this merged.

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "repository": "pillarjs/hbs",
   "main": "lib/hbs.js",
   "dependencies": {
-    "handlebars": "4.3.5",
+    "handlebars": "4.5.3",
```
Comment (blakeembrey) [inline package.json]: @dougwilson Would it make sense to pin it to the major version so people can get updates independently? Alternatively, change it to a peer dependency?

Comment (dougwilson) [reply package.json]: Handlebars has not been following semver and there have been several non major releases where parts of this module had to be changed in order to work with the new handlebars.

File: HISTORY.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+unreleased
+==================
+  * deps: handlebars@4.5.3
+    - Fix security issues
```
Comment (dougwilson) [inline HISTORY.md]: This change list seems incomplete; there are more changes than just this in the bump.

File: HISTORY.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+unreleased
+==================
```
Comment (dougwilson) [inline HISTORY.md]: The formatting is messed up here: just shorten the line and add a blank line after this and before the bullet point.

File: package.json (hunk 1 above)
Comment (blakeembrey) [reply package.json]: Ah, that's unfortunate - thanks!

Comment (dougwilson) [reply package.json]: Yea. It seems to be common among libs that started out as client side and then just publish to npm. jQuery, lodash, etc. all seem similar in that regard as well.

**Pull request closed without merging**